sstable_generation_loading_test.py

```diff
@@ -35,10 +35,10 @@ def compact(self):
         return self.fixture_dtest_setup.cluster.version() < MAJOR_VERSION_4 and self.test_compact
 
     def create_schema(self, session, ks, compression):
+        legacy_compression_class = self.fixture_dtest_setup.cluster.version() < MAJOR_VERSION_3
```

Comment:
The cassandra-18742 patch is fixed as of 5.0, so the obsolete param should work for versions < 5.0, right? I wonder if we can fix this condition...

Reply:
@Mmuzaf  yes, you are technically right, it will work with `sstable_compression` until 5.0 but I am thinking .... why not to use non-deprecated param already for all branches which support it?

Reply:
yes, but in this case it is not clear from which version the parameter was removed :-(